tests/integration/test_manylinux.py

```diff
@@ -348,8 +348,9 @@ def test_build_repair_pure_wheel(any_manylinux_container, io_folder):
     ]) in output.replace('\n', ' ')
 
 
+@pytest.mark.parametrize('dtag', ['rpath', 'runpath'])
 def test_build_wheel_depending_on_library_with_rpath(any_manylinux_container, docker_python,
-                                                     io_folder):
+                                                     io_folder, dtag):
     # Test building a wheel that contains an extension depending on a library with RPATH set
```

Comment:
Given that this is quite a complicated test I think it would be nice to have a few words on the additional checks on `RPATH`/`RUNPATH` you are adding now, to make it a bit easier to understand the code.